virttest/qemu_storage.py

```diff
@@ -828,7 +828,7 @@ def compare_to(self, target_image, source_cache_mode=None,
 
         return result
 
-    def check_image(self, params, root_dir, force_share=False):
+    def check_image(self, params, root_dir, force_share=False, output="human"):
```

Comment:
Please add the param to the docstring.

Reply:
Also, I believe it is better to set the default to `None`.

Reply:
Updated, thanks.